map.json

```diff
@@ -0,0 +1,155 @@
+{
+  ".github": [],
+  "documentation": [],
+  "setup": [
+    ".\\tools\\StaticAnalysis\\StaticAnalysis.Test\\bin\\Debug\\StaticAnalysis.Test.dll"
+  ],
+  "src/Common": [
+    ".\\src\\Common\\Commands.Common.Authentication.Test\\bin\\Debug\\Microsoft.Azure.Commands.Common.Authentication.Test.dll"
```

Comment:
changes to src\common should result in all tests being executed.  Need to make sure this assembly is included (perhaps by including the test directory as the key (src/Common/Commands.Common.Authentication.Test)